app/controllers/v1/bookings_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module V1
+  class BookingsController < ApplicationController
+    before_action :set_meeting_room, only: [:create]
+    before_action :set_user, only: [:create]
+
+    def create
+      # Check whether Meeting Room is available.
+      unless @meeting_room.is_available?
+        return render json: { error: 'Sorry, Meeting Room not available for booking.' }, status: :ok
+      end
```

Comment:
@syedtabrez123  You are checking availability of meeting so I think after booking the meeting you should make is_available = false for this meeting. Correct me If I am wrong.

Reply:
Nope, If all time slots are booked then we need to make is_available false, not after only one booking. I hope it clarifies. @nidhi-ranjan @glen

Reply:
@syedtabrez123 yes , I got it.

Reply:
the  `is_available` flag is for the meeting room - to indicate that the meeting room is available to be booked or is temporarily out of service (think of it as a leakage - and hence that room cannot be used).

Reply:
Yes @nidhi-ranjan as of now the purpose of is_available flag it to indicate availability meeting room as @glen . I think in future we can use same the flag if all time slots are booked for the day.